pySDC/implementations/problem_classes/generic_ND_FD.py

```diff
@@ -16,6 +16,48 @@
 
 
 class GenericNDimFinDiff(ptype):
+    r"""
+    Example implementing the N-dimensional linear equation
+
+    .. math::
+        \frac{d u}{dt} = A u,
+
+    where :math:`A  \in \mathbb{R}^{n \times n}` is a finite difference matrix arising from spatial discretization.
```

Comment:
where :math:`A \in \mathbb{R}^{nN \times nN} is a matrix arising from finite difference discretisation of spatial derivatives with n degrees of freedom per dimension and N dimensions.